golem-worker-executor-base/tests/wasi.rs

```diff
@@ -1849,3 +1849,47 @@ async fn ip_address_resolve(
     check!(result1.len() > 0);
     check!(result2.len() > 0);
 }
+
+#[test]
+#[tracing::instrument]
+async fn wasi_incoming_request_handler_compat(
```

Comment:
better test coverage please! :) (headers, body, etc)

Reply:
Yes, this is coming :+1:

Reply:
done